walletnode/examples/configs/localnet.yml

```diff
@@ -5,29 +5,21 @@ temp-dir: "/tmp/walletnode"
 work-dir:
 rq-files-dir: 
 
+log-level:
+log-file:
+quiet: false
+temp-dir:
+work-dir:
+rq-files-dir: 
```

Comment:
these last two are in the lines 5 and 6 above too

Reply:
ah, good catch! Thank you